ts/src/header-validator/trigger.test.ts

```diff
@@ -19,6 +19,11 @@ const testCases: jsontest.TestCase<Trigger>[] = [
         "filters": {"x": []},
         "not_filters": {"y": []}
       }],
+      "aggregatable_buckets": [{
+        "bucket": "bucket",
```

Comment:
Let's give the value a different string so it's clear that it's different from the key.